src/schema.graphql

```diff
@@ -63,6 +63,8 @@ type Profile {
   titleFr: String
   team: Team
   ownerOfTeams: [Team!]!
+  outstandingApprovals: [Approval!]!
+  submittedApprovals: [Approval!]!
```

Comment:
Needed to add the 2 fields for approver and submitter on the Profile schema because of the @relation field defined on the Approval schema